demo/api/src/brevo-contact/dto/brevo-contact-subscribe.input.ts

```diff
@@ -0,0 +1,25 @@
+import { IsValidRedirectURLConstraint } from "@comet/brevo-api";
+import { Type } from "class-transformer";
+import { IsEmail, IsNotEmpty, IsUrl, Validate, ValidateNested } from "class-validator";
+
+import { BrevoContactAttributes } from "./brevo-contact-attributes";
+import { EmailContactSubscribeScope } from "./brevo-contact-subscribe.scope";
+
+export class BrevoContactSubscribeInput {
+    @IsEmail()
+    email: string;
+
+    @IsUrl({ require_tld: process.env.NODE_ENV === "production" })
+    @Validate(IsValidRedirectURLConstraint)
```

Comment:
Shouldn't this be something like `@IsValidRedirectURL()`?

Reply:
yes, thank you.
copy paste error, I'll update it in the other places as well in an extra PR.